### PR TITLE
Added short description how configure DNS Zones

### PR DESCRIPTION
**Pull request by kamil-mrzyglod**
I added a short description how to configure Azure DNS Zones when using subdomains and Azure App Service on Linux.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,7 +107,27 @@ If they need a Wildcard certificate, additional assign `DNS Zone Contributor` ro
...
+* api.contoso.com
+
+You need to create 3 Azure DNS Zones:
+* contoso.com
+* web.contoso.com
+* api.contoso.com
+
+After delegating a domain to Azure DNS Zone(using Name servers attached to the contoso.com zone), you will need to add to contoso.com DNS Zone the following entries:
+* web(Type: NS, Value: Name servers attached to web.contoso.com Azure DNS Zone)
+* api(Type: NS, Value: Name servers attached to api.contoso.com Azure DNS Zone)
```
Comment (shibayan) [inline README.md]: In this case, only the `contoso.com` should have an Azure DNS zone.
Maybe it didn't work?

Comment (kamil-mrzyglod) [reply README.md]: Hey, in my case I was required to provide two additional Azure DNS Zones named exactly as domains attached to them. Failing to do so results in error reported that e.g. web.contoso.com Azure DNS Zone was not found.

Comment (shibayan) [reply README.md]: That is not my intended behavior. Please provide information as there is a possibility of a bug.

Comment (kamil-mrzyglod) [reply README.md]: Hey, so the setup is as follows:
* front-end app(web)
* back-end app(api)

Both have custom domains configured. Both are hosted with separate Web Apps. While requesting for a certificate, custom domains are correctly discovered, however using them to obtain a cert results in "Cannot find Azure DNS Zone". In the logs I saw it searched for web.* and api.* DNS Zones, not the main domain.

**Pull request closed without merging**